arch/arm/src/tms570/Make.defs

```diff
@@ -23,7 +23,7 @@ include armv7-r/Make.defs
 # SAMA5-specific C source files
 
 CHIP_CSRCS  = tms570_boot.c tms570_clockconfig.c tms570_esm.c tms570_gio.c
-CHIP_CSRCS += tms570_irq.c tms570_lowputc.c tms570_serial.c tms570_spi.c
```

Comment:
Why to remove tms570_spi.c ?

Reply:
Ok, I saw the issue. I think @patacongo added support to SPI on TMS570 but missed to include the tms570_spi.c inside arch/, there is only tms570_spi.c inside board. Initially I thought someone had removed this file, but then I searched on all my NuttX backup files and didn't find it.